source/neuropod/bindings/c/c_api.h

```diff
@@ -23,25 +23,52 @@ limitations under the License.
 #include "neuropod/bindings/c/np_tensor_spec.h"
 #include "neuropod/bindings/c/np_valuemap.h"
 
+#include <stdbool.h>
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 typedef struct NP_Neuropod NP_Neuropod;
 
-// Load a model given a path
+// Load a model given a path.
+// status is required. Note that function sets it without the check on nullptr.
```

Comment:
Not sure if we need this comment here and below.

In the context of our back and forth, I think it makes sense (because we made a change related to that functionality), but I think it's a little odd to include it in the final code especially since we don't document this for every param of the C API functions

Reply:
I will remove comment with status for now. We may think about some standard for description of public API, like Docygen or other and strictly follow it.

Reply:
Discussed offline https://google.github.io/styleguide/cppguide.html#Function_Comments